infra-core/src/main/java/com/redhat/service/smartevents/infra/core/models/connectors/ConnectorType.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.service.smartevents.infra.models.connectors;
+package com.redhat.service.smartevents.infra.core.models.connectors;
```

Comment:
what about moving the connectors to `v1`?

Reply:
Agreed. Done.